content/post/Android_Root101.en.md

```diff
@@ -73,7 +76,7 @@ Moving on.
 
 > type-in `fastboot boot <nameOfTheRecoveryFile.img>`
```

Comment:
and here, 
it is a technical blog,
`type` is not appropriate, this is not a kid's tutorial